xbmc/guilib/GUIMoverControl.cpp

```diff
@@ -257,7 +257,9 @@ void CGUIMoverControl::SetPosition(float posX, float posY)
 
 bool CGUIMoverControl::SetAlpha(unsigned char alpha)
 {
-  return m_imgFocus->SetAlpha(alpha) | m_imgNoFocus->SetAlpha(alpha);
```

Comment:
What about logical `operator||` ?

Reply:
The logical `||` operator would change behaviour (short circuit).

Reply:
What is the intended behaviour?

Reply:
Always call both functions and return true if at least one function returned true.
If I understand the code correctly the return value indicates if `MarkDirtyRegion()` has to be called.